src/site/includes/header.html

```diff
@@ -64,6 +64,7 @@
 
 <script type="text/javascript">
   window.VetsGov = window.VetsGov || {};
+  window.featureToggles = {{ featureToggles }};
```

Comment:
some metalsmith magic here- `featureToggles` is available to the template from the `createFeatureToggles` plugin